src/main/java/com/minecolonies/permissions/ColonyPermissionEventHandler.java

```diff
@@ -0,0 +1,240 @@
+package com.minecolonies.permissions;
+
+import com.minecolonies.blocks.AbstractBlockHut;
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.permissions.Permissions;
+import com.minecolonies.util.Log;
+import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.event.entity.item.ItemTossEvent;
+import net.minecraftforge.event.entity.player.PlayerContainerEvent;
+import net.minecraftforge.event.entity.player.PlayerInteractEvent;
+import net.minecraftforge.event.world.BlockEvent;
+import net.minecraftforge.event.world.ExplosionEvent;
+import net.minecraftforge.fml.common.eventhandler.Event;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+/**
+ *
+ */
+public class ColonyPermissionEventHandler
+{
+
+    private final Colony colony;
+
+    public ColonyPermissionEventHandler(final Colony colony)
+    {
+        this.colony = colony;
+    }
+
+    /**
+     * BlockEvent.PlaceEvent handler.
+     *
+     * @param event BlockEvent.PlaceEvent
+     */
+    @SubscribeEvent
+    public void on(final BlockEvent.PlaceEvent event)
+    {
+        if (checkBlockEventDenied(event.getWorld(), event.getPos(), event.getPlayer(), event.getPlacedBlock())) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)